libraries/chain/controller.cpp

```diff
@@ -3954,7 +3954,7 @@ struct controller_impl {
 
             // if previous block's header extension has the same claim, just return
             // (previous block already validated the claim)
-            return {};
+            return std::optional<qc_t>{};
```

Comment:
I don't think this change is warranted. If we were returning a value in the optional, yes, but here we return an empty optional so there is no extra object construction.